privacy-protections/fingerprinting/index.html

```diff
@@ -12,14 +12,15 @@
     <link href='./style.css' rel='stylesheet'></link>
 </head>
 <body>
-    <p><a href="../../">[Home]</a> ↣ <a href="../">[Privacy Protections Tests]</a></p>
+    <p><a href="../../">[Home]</a> ↣ <a href="../">[Privacy Protections Tests]</a> ↣ <strong>[Fingerprinting Test Page]</strong></p>
 
     <p>This page collects various datapoints from your browser that can be used for fingerprinting.</p>
     <p>⚠️ Please note that: </p>
     <ul>
         <li>not all of the properties listed here have to be modified to succesfully prevent fingerprining,</li>
-        <li>some properties may change randomly, depending on the brower, on every page reload or test run,</li>
-        <li>some data is duplicated, but it's retrieved using different methods.</li>
+        <li>depending on the brower some properties may change randomly on every page reload or test run,</li>
+        <li>some data is duplicated, but it's retrieved using different methods,</li>
+        <li>this page has to be loaded over https for all tests to work correctly (some APIs are not available over http).</li>
```

Comment:
Since glitch is supporting both http and https I added a note that fingerprinting page should be loaded over https.